pkgs/development/libraries/WavePack/default.nix

```
@@ -0,0 +1,21 @@
+{ stdenv, autoreconfHook, fetchgit }:
+stdenv.mkDerivation rec {
+  name = "WavePack-${version}";
```

```suggestion
  name = "wavpack-${version}";
```

No 'e' in upstream name and no caps as per manual guidelines:
https://nixos.org/nixpkgs/manual/#sec-package-naming

Also correct the name of the top directory

---

"wavpack" ends up already in nixpkgs. I'll drop this commit then.